src/background.js

```diff
@@ -47,14 +46,11 @@ class containersTheme {
 
   isUnpaintedTheme(currentCookieStore) {
     return (currentCookieStore == "firefox-default" ||
-            currentCookieStore == "firefox-private");
+      currentCookieStore == "firefox-private");
   }
 
-  resetTheme() {
-    // Because of the following, we loop through all active windows after a reset
```

Comment:
Should this comment move to above, new line 33 (else clause)? 

If I'm reading the code correctly, we now invoke resetTheme for every unpainted active tab, which _might_ be more often.  I'll try on my machine later today -- I have enough open tabs that I should be able to notice flicker. :)